README.md

```diff
@@ -1 +1,34 @@
-Readme
+<a href="https://www.pkgup.io/"><img src="https://user-images.githubusercontent.com/14764799/135774154-6544ee1e-0ce1-4a38-96b3-546149213c31.png" height="90" /></a><br />
+_Package Updater simplifies your development workflow by helping you smartly update your project's npm packages._
+
+# Package Updater -  App Support
+
+If you are reporting a bug, please add detailed steps to reproduce the bug, the Package Updater version you're using, and your operating system (including the OS version). Adding additional files (console errors, screenshots, or other content) would help us better assist you by identifying the target issue. 
+
+> **Account Specific Queries:**
+> If you have any billing or account-specific queries, reach out to us by submitting a ticket on our [support page](https://www.pkgup.io/contact/).
+
+## Guidelines for reporting issues 
+
+Below is a short guideline to follow when reporting an issue. Following the guidline will help resolve issues in a timely manner. 
+
+1. If you are facing an account-related issue and you want to include personal information such as your username or collection names, you can reach out to us on our [support page](https://www.pkgup.io/contact/). 
+
+2. When reporting a new issue, it helps to answer the following questions:
```

Comment:
We can don't add it to readme, later I'll add the issue template

Reply:
Ok deleted